src/Stratis.Bitcoin/P2P/RelatedPeerConnectors.cs

```diff
@@ -4,9 +4,9 @@
 
 namespace Stratis.Bitcoin.P2P
 {
-    public sealed class RelatedPeerConnectors : Dictionary<string, PeerConnector>
+    public sealed class RelatedPeerConnectors : Dictionary<string, IPeerConnector>
```

Comment:
Ah ok this is just to introduce the `IPeerConnector` a future PR will get ride of `RelatedPeerConnectors ` thats fine